presentation/build.gradle.kts

```diff
@@ -40,32 +41,33 @@ android {
 }
 
 dependencies {
-  val composeVersion = "1.4.0-alpha04"
-  val coilVersion = "2.2.2"
+  val composeBom = platform(libs.androidx.compose.bom)
```

Comment:
I added in these couple lines from the official documentation to make use of the BOM.